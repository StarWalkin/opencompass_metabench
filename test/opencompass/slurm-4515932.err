  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4515932 ON gpua012 CANCELLED AT 2024-08-18T00:18:06 ***
