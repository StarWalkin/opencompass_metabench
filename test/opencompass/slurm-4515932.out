/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/18 00:07:52 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/18 00:07:52 - OpenCompass - INFO - Loading hf_phi_3_small_8k_instruct: configs/models/phi/hf_phi_3_small_8k_instruct.py
08/18 00:07:52 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/18 00:07:53 - OpenCompass - INFO - Current exp folder: outputs/default/20240818_000753
08/18 00:07:53 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/18 00:07:53 - OpenCompass - INFO - Partitioned into 1 tasks.
